pom.xml

```diff
@@ -135,7 +135,7 @@
     <jetty.version>9.4.11.v20180605</jetty.version>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <maven.plugin-version>3.2.5</maven.plugin-version>
+    <maven.plugin-version>3.5.4</maven.plugin-version>
```

Comment:
Maven plugin dependencies are in a somewhat strange situation, different from other dependencies. That's because they are used inside of Maven's own build execution environment, and it's not clear to me from the limited Maven documentation which of these dependencies are "provided" by Maven, and whether this will result in restricting projects who use `accumulo-maven-plugin` from being able to do a build in earlier versions of Maven.

I think this change is okay... but we should pay attention to users who might report that they are unable to perform a build using an older version of Maven.